content/bayes_nn/bayes.md

```diff
@@ -0,0 +1,318 @@
+# Bayesian Neural Network
+
+The usual Neural Network are optimized in way to get fixed value of weights and biases that allows the model perform specific task successfully. Instead in 
+Bayesian Neural Network the weights and biases are the distribution, this type of model could be treated as a ensemble of many neural networks trained by the Bayesian inference.
+
+Bayesian approach for the neural networks allows to estimate the uncertainty and make the decision of the model more robust according to the input data.
+
+
+
+### Difference between usual NN and BNN
+
+
+![Placeholder](../images/bayes_nn/diff.png)
+
+
+### Training of NN and BNN
+=== "NN"
+    ![Placeholder](../images/bayes_nn/trainingNN.png)
+    The parameters  ![formula](https://render.githubusercontent.com/render/math?math=\theta ) are optimized in order to minimaze the loss function
+
+=== "BNN"
+    ![Placeholder](../images/bayes_nn/bayesNN.png)
+     The process is to learn the probability distributions for weights and biases that maximize the likelihood of getting a high probability for the correct data/label ![formula](https://render.githubusercontent.com/render/math?math=D(x,y) ) pairs.
+    The parameters of the weights distributions - mean and standart deviation are the product of the loss function optimization
+
+#### Training Procedure
+    1. Introduce the prior distribution over model parameter w
+    2. Compute posterio p(w|D) using Bayesian rule
+    3. Make the average over the posterior distribution
+
+
+
+### Prediction of NN and BNN
+=== "NN"
+    ![Placeholder](../images/bayes_nn/PredictionNN.png)
+
+=== "BNN"
+    ![Placeholder](../images/bayes_nn/PredictionBNN.png)
+
+
+### Uncertainty
+The uncertainty that are quatified by the BNN are categorized in the next way:
```

Comment:
"There are two types of BNN uncertainties:
1. Alletonic - uncertainties due to the lack of knowledge, comes from data or environment ![formula](https://render.githubusercontent.com/render/math?math=p (\theta|D) )
2. Epistemic - uncertainties of the model parameter ![formula](https://render.githubusercontent.com/render/math?math=p(y|x,\theta))
"

Also, the formulas are not being rendered.